package.json

```diff
@@ -7,17 +7,20 @@
     "url": "https://github.com/openfisca/openfisca-doc.git"
   },
   "scripts": {
+    "prebuild": "npm run clean",
     "build": "gitbook build",
-    "clean": "rm -rf _book",
+    "clean": "del _book",
+    "deploy": "npm run build && npm run deploy:rsync && git fetch",
```

Comment:
I'm ok with keeping this command, but I maintain my first question : why do we fetch at the end of a deployment ?